cura/API/Account.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) 2018 Ultimaker B.V.
+# Cura is released under the terms of the LGPLv3 or higher.
+from typing import Optional, Dict
+
+from PyQt5.QtCore import QObject, pyqtSignal, pyqtSlot, pyqtProperty
+
+from UM.Message import Message
+from cura.OAuth2.AuthorizationService import AuthorizationService
+from cura.OAuth2.Models import OAuth2Settings
+from UM.Application import Application
+
+from UM.i18n import i18nCatalog
+i18n_catalog = i18nCatalog("cura")
+
+
+##  The account API provides a version-proof bridge to use Ultimaker Accounts
+#
+#   Usage:
+#       ``from cura.API import CuraAPI
+#       api = CuraAPI()
+#       api.account.login()
+#       api.account.logout()
+#       api.account.userProfile # Who is logged in``
+#
+class Account(QObject):
+    # Signal emitted when user logged in or out.
+    loginStateChanged = pyqtSignal()
```

Comment:
Does this signal have a callback argument that passes the new login state? The backups plugin relies on this to see if we need to fetch the backups after logging in.

Reply:
I can add it, but this is code I copied from the plugin. Account does have a isLoggedIn property though. 
So let me know if you need it, it's easy enough to add.

Reply:
Uhg. i need to read better. You already made a pull request and that one uses it. Fixing it.